### PR TITLE
opendal: fix environment problem

### PR DESCRIPTION
**Pull request by dqhl76**
Fix the problem that opendal cannot init service
related PR on opendal side: https://github.com/apache/incubator-opendal/pull/2824
/cc @Xuanwo

### CODE REVIEW

File: projects/opendal/build.sh [hunk 1]
```diff
@@ -25,6 +22,12 @@ targets=(
...
-  cp ../target/x86_64-unknown-linux-gnu/release/$target $OUT/
-done
+
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_reader_fs
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_writer_fs
+
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_reader_memory
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_writer_memory
+
+echo -e "OPENDAL_FS_TEST=on\nOPENDAL_FS_ROOT=/tmp" > $OUT/.fs.env
```
Comment (Xuanwo) [inline projects/opendal/build.sh:32]: I prefer to create `.fs.env` by here doc:

```shell
cat <<EOF > $OUT/.fs.env
OPENDAL_FS_TEST=on
OPENDAL_FS_ROOT=/tmp
EOF
```

I find this way is easy to maintain.

File: projects/opendal/build.sh [hunk 2]
```diff
@@ -25,6 +22,12 @@ targets=(
   fuzz_reader
   fuzz_writer
 )
-for target in "${targets[@]}"; do
-  cp ../target/x86_64-unknown-linux-gnu/release/$target $OUT/
-done
+
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_reader_fs
```
Comment (Xuanwo) [inline projects/opendal/build.sh:26]: I prefer to organize code in service order, for exemaple:

- create fs env
- copy files to fuzz-reader_fs

**Comment by Xuanwo** [comment]
Maybe we can commit `.fs.env` directly and just copy them?

**Comment by dqhl76** [comment]
Sure. Reorganized the files!

**Review by Xuanwo — APPROVED** [review]
LGTM

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**